doc/source/release/1.7.0-notes.rst

```diff
@@ -162,7 +162,7 @@ Added experimental support for the AArch64 architecture.
 C API
 -----
 
-New function ``PyArray_RequireWriteable`` provides a consistent interface
+New function ``PyArray_FailUnlessWriteable`` provides a consistent interface
```

Comment:
For other reviewers who are confused like I was, this is really a typo in the 1.7.0 release note. PR #280, which was part of 1.7, renamed the function before its first release, and somehow the release note was not updated.